modules/GenericSensor.js

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2016, Intel Corporation.
```

Comment:
Please update copyrights. 2016-2017 for this one that reuses some old code. 2017 for other files that are all new.